packages/web/config/chain-infos.ts

```diff
@@ -2064,28 +2065,57 @@ const chainInfos = (
       explorerUrlToTx: "https://explorer.teritori.com/teritori/tx/{txHash}",
     },
     {
-        rpc: "https://rpc.lambda.im",
-        rest: "https://rest.lambda.im",
-        chainId: "lambda_92000-1",
-        chainName: "Lambda",
-        bip44: {
-          coinType: 60,
-        },
-        bech32Config: Bech32Address.defaultBech32Config("lamb"),
-        currencies: [
-          {
-            coinDenom: "LAMB",
-            coinMinimalDenom: "ulamb",
-            coinDecimals: 18,
-            coinGeckoId: "pool:lambda",
-            coinImageUrl: "/tokens/lambda.png",
-            isStakeCurrency: true,
-            isFeeCurrency: true,
-          },
-        ],
-        features: ["ibc-transfer", "ibc-go", "eth-address-gen", "eth-key-sign"],
-        explorerUrlToTx: "https://explorer.nodestake.top/lambda/tx/{txHash}",
+      rpc: "https://rpc.lambda.im",
+      rest: "https://rest.lambda.im",
+      chainId: "lambda_92000-1",
+      chainName: "Lambda",
+      bip44: {
+        coinType: 60,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("lamb"),
+      currencies: [
+        {
+          coinDenom: "LAMB",
+          coinMinimalDenom: "ulamb",
+          coinDecimals: 18,
+          coinGeckoId: "pool:lambda",
+          coinImageUrl: "/tokens/lambda.png",
+          isStakeCurrency: true,
+          isFeeCurrency: true,
+        },
+      ],
+      features: ["ibc-transfer", "ibc-go", "eth-address-gen", "eth-key-sign"],
+      explorerUrlToTx: "https://explorer.nodestake.top/lambda/tx/{txHash}",
+    },
+    {
+      rpc: "https://rpc.unification.chainmasters.ninja",
+      rest: "https://rest.unification.chainmasters.ninja",
+      chainId: "FUND-MainNet-2",
+      chainName: "Unification",
+      bip44: {
+        coinType: 5555,
+      },
+      bech32Config: Bech32Address.defaultBech32Config("und"),
+      currencies: [
+        {
+          coinDenom: "FUND",
+          coinMinimalDenom: "nund",
+          coinDecimals: 9,
+          coinGeckoId: "unification",
+          coinImageUrl: "/tokens/fund.png",
```

Comment:
Please upload this image to the tokens folder

Reply:
Done